x-pack/plugins/infra/common/log_views/defaults.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { defaultSourceConfiguration } from '../source_configuration/defaults';
+import { LogViewAttributes, LogViewsStaticConfig } from './types';
+
+export const defaultLogViewAttributes: LogViewAttributes = {
+  name: 'Log View',
+  description: 'A default log view',
+  logIndices: {
+    type: 'index_name',
+    indexName: 'logs-*,filebeat-*',
+  },
+  logColumns: [
+    {
+      timestampColumn: {
+        id: '5e7f964a-be8a-40d8-88d2-fbcfbdca0e2f',
```

Comment:
I statically generated them when implementing the columns :game_die: Their purpose is to

1. improve rendering performance because we have a stable `key`
2. allow detection of whether a user has manually changed a column if we want to apply an automatic saved object migration later (e.g. replace the reconstructed message column type with the message field column type)